integration/helpers.go

```diff
@@ -93,15 +93,19 @@ out:
 	return nil
 }
 
+func BuildCommonData(namespace string) eventtypes.CommonData {
+	return eventtypes.CommonData{
+		Namespace: namespace,
+		// Pod and Container name are defined by BusyboxPodCommand.
+		Pod:       "test-pod",
+		Container: "test-pod",
+		// TODO: Include the Node
+	}
+}
+
 func BuildBaseEvent(namespace string) eventtypes.Event {
 	return eventtypes.Event{
-		Type: eventtypes.NORMAL,
-		CommonData: eventtypes.CommonData{
-			Namespace: namespace,
-			// Pod and Container name are defined by BusyboxPodCommand.
-			Pod:       "test-pod",
-			Container: "test-pod",
-			// TODO: Include the Node
-		},
+		Type:       eventtypes.NORMAL,
+		CommonData: BuildCommonData(namespace),
 	}
 }
```

Comment:
Hey @mauriciovasquezbernal, I will create an issue to organize these structures so that we avoid having the top gadgets defining their own [Error](https://github.com/inspektor-gadget/inspektor-gadget/blob/0ca2f8466f8f27d5fd627fb20c6c91cfc78ad931/pkg/gadgets/top/tcp/types/types.go#L86) field and error mechanism, instead of using [these](https://github.com/inspektor-gadget/inspektor-gadget/blob/0ca2f8466f8f27d5fd627fb20c6c91cfc78ad931/pkg/types/types.go#L85-L89).

Reply:
Sure, makes sense.